content/news/2023-10-09-positconf-chicago-meetup/index.en.md

```diff
@@ -0,0 +1,217 @@
+there are things we can think about to make it more inclusive. Things
+like having your space easily accessible: in Oslo we follow Carpentries rules
+about where the talks are, for example it should be wheelchair
+accessible, and near public transportation, not too late in the evening
+so they are not falling asleep and not too early to account for
+children. Allowing kids is another option chapter organizers can decide
+for their communities, and they have the freedom to decide how and what
+you think is best. If anyone has tips or ideas the group is open to
+suggestions. We have the R-Ladies guide and there are general
+recommendations; please reach out to us with new ideas so we can share
+with the global community.
+
+## What do we think about the idea of having a RLadies conference day? 👀😊
+
+*Mo:* That is the dream! We are not there yet, and the leadership needs
+to be more established, but it is definitely something to consider.
+
+# Operational/procedural questions
+
+## What are chapter organizers finding helpful as we go back in person?
+
+*María:* Online meetings have been helpful to reach other people that we
+haven't previously been able to and in other parts of Latin America; we
+plan to keep running online meet ups in a 50/50 way. I am glad to be
+back in person also to meet people and connect in a different way.
+
+*Stephanie:* I have had similar experience; we have people from all
+different universities and working in different sectors attending our events. Because it
+is very difficult to move around city after work, we had talked about
+virtual even before the pandemic, because we wanted to reach a wider audience.
+Now most of our events are virtual to be more inclusive and make it
+easier for people to attend events.
+
+*Angela:* Online events allow for the possibility of collaborations with
+other chapters that didn't exist before. Additionally, before there were
+a lot of details in terms of having to find space, get pizza, and open
+doors, now with Zoom this is less so the case now and it makes things
+easier on the organizers and those people that are coming to speak.
+
+## What is the process of starting a chapter?
+
+*Riva:* Send an email to [chapters\@rladies.org](mailto:chapters@rladies.org). A member of the Onboarding Team will welcome you
+and explain process. Chapters are permitted for cities, but not for a
+university or company. If you wish to have a thematic meet-up, you could
+approach a chapter like R-Ladies Remote and they can help create
+thematic meet-ups.
+
+## How would I step up and take over an inactive local chapter?
+
+*Riva:* Same process; email
+[chapters\@rladies.org](mailto:chapters@rladies.org), and they would be
+happy to begin the process to reactivate your chapter.
+
+## If I don't have a local chapter, is there a way to join a "virtual" chapter? Or are there chapters that regularly meet online?
+
+The R-Ladies Remote chapter serves this purpose. Additionally, many
```

Comment:
We don't have a speaker listed here - Mo or Riva?

Reply:
I think this was both Mo and Riva who spoke on this question.